src/Promenade.Web/Views/Shared/_Layout.cshtml

```diff
@@ -336,15 +400,27 @@
                         @foreach (var item in footerNavigation.Navigations)
                         {
                             <div>
-                                <a href="@item.Link"
-                                   aria-label="@item.NavigationText.AriaLabel"
-                                   class="@(item.NavigationText.Label == null ? "" : "d-flex align-items-center")">
-                                    @if (!string.IsNullOrEmpty(item.Icon))
-                                    {
-                                        <span class="@item.Icon @(item.NavigationText.Label == null ? "" : "prom-fa-space")"></span>
-                                    }
-                                    <span class="prom-footer-link-text">@item.NavigationText.Label</span>
-                                </a>
+                                @if (string.IsNullOrEmpty(item.NavigationText.Label)
+                                    && !string.IsNullOrEmpty(item.Icon)
+                                    && !string.IsNullOrEmpty(item.NavigationText.Title))
+                                {
+                                    <a href="@item.Link"
+                                       title="@item.NavigationText.Title">
+                                        <span class="@item.Icon></span>
+                                        <span class="prom-footer-link-text">@item.NavigationText.Label</span>
```

Comment:
This span can be removed since the label will be empty.

Reply:
Resolved in commit 9b9fd46116bc0d3399fe1289b3f3852d4fd5576c